cli/src/main/java/com/liferay/blade/cli/Extensions.java

```diff
@@ -85,9 +87,19 @@
 	}
 
 	public static Path getDirectory() {
-		try {
-			Path userHomePath = BladeCLI.USER_HOME_DIR.toPath();
+		return getDirectory(Paths.get(System.getProperty("user.home")));
+	}
+
+	public static Path getDirectory(BladeCLI blade) {
```

Comment:
Since Extensions is instantiated, just pass in the userHomeDirectory to it as well as BladeSettings, that way it doesn't need this method.